doc/topics/releases/oxygen.rst

```diff
@@ -51,6 +51,12 @@ New NaCl Renderer
 
 A new renderer has been added for encrypted data.
 
+New salt-ssh roster
+-------------------
+
+A new roster has been added that allows users to pull in a list of hosts
```

Comment:
It would be good to mention what the name of the new roster is so people can refer to those docs on how to use it.

Reply:
Fixed.